account_due_list_aging_comments/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Account Due List Aging Comments
+===============================
+
+This module extends the Payments and Due list to add the capability to
+introduce aging comments in-line.
+
+Usage
+=====
+
+To use this module, you need to go to:
+
+* Accounting->Journal Entries->Payments and due list
```

Comment:
@jbeficent `account_due_list` needs to depend on `account_accountant` to have access to the Adviser submenu (see #135)